plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -54,15 +54,15 @@ public class HudiSessionProperties
     private static final String MAX_SPLITS_PER_SECOND = "max_splits_per_second";
     private static final String MAX_OUTSTANDING_SPLITS = "max_outstanding_splits";
     private static final String SPLIT_GENERATOR_PARALLELISM = "split_generator_parallelism";
-    static final String QUERY_PARTITION_FILTER_REQUIRED = "query_partition_filter_required";
+    public static final String QUERY_PARTITION_FILTER_REQUIRED = "query_partition_filter_required";
     private static final String IGNORE_ABSENT_PARTITIONS = "ignore_absent_partitions";
-    private static final String DYNAMIC_FILTERING_WAIT_TIMEOUT = "dynamic_filtering_wait_timeout";
+    public static final String DYNAMIC_FILTERING_WAIT_TIMEOUT = "dynamic_filtering_wait_timeout";
 
     // Internal configuration for debugging and testing
-    static final String RECORD_LEVEL_INDEX_ENABLED = "record_level_index_enabled";
-    static final String SECONDARY_INDEX_ENABLED = "secondary_index_enabled";
-    static final String COLUMN_STATS_INDEX_ENABLED = "column_stats_index_enabled";
-    static final String PARTITION_STATS_INDEX_ENABLED = "partition_stats_index_enabled";
+    public static final String RECORD_LEVEL_INDEX_ENABLED = "record_level_index_enabled";
+    public static final String SECONDARY_INDEX_ENABLED = "secondary_index_enabled";
+    public static final String COLUMN_STATS_INDEX_ENABLED = "column_stats_index_enabled";
+    public static final String PARTITION_STATS_INDEX_ENABLED = "partition_stats_index_enabled";
```

Comment:
Does these have to be public?  I see the modified test classes are in the same package.